### PR TITLE
ButtonWidget: block alignment

### PR DESCRIPTION
**Pull request by apepper**
This PR adds a fourth "alignment": "Block level"

block level

### CODE REVIEW

File: src/Widgets/ButtonWidget/ButtonWidgetEditingConfig.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ Scrivito.provideEditingConfig("ButtonWidget", {
         { value: "left", title: "Left" },
         { value: "center", title: "Center" },
         { value: "right", title: "Right" },
+        { value: "block", title: "Block level" },
```
Comment (dcsaszar) [inline src/Widgets/ButtonWidget/ButtonWidgetEditingConfig.js:15]: * [ ] FYI In the Bootstrap docs there is a technical term and a human readable term:
> block level buttons—those that span the full width of a parent

I think we should prefer the technical term on the tech side and the human term on the editor side:
```ts
{ value: "block", title: "Full width" },
```

File: src/Widgets/ButtonWidget/ButtonWidgetComponent.js [hunk 2]
```diff
@@ -17,21 +17,19 @@ const ButtonWidgetComponent = Scrivito.connect(({ widget }) => {
...
-    return (
-      <div className={`text-${widget.get("alignment")}`}>
-        <ButtonWidgetComponent widget={widget} />
-      </div>
-    );
-  }
-
-  return <ButtonWidgetComponent widget={widget} />;
-});
+function AlignmentWrapper({ alignment, children }) {
```
Comment (dcsaszar) [inline src/Widgets/ButtonWidget/ButtonWidgetComponent.js:33]: * [ ] FYI Looking at the next commit it becomes clear that the `alignment` setting is not exclusively evaluated here, which makes the naming slightly off. How about `AlignedText`, `TextAlignment`, `TextWrapper` or so?

**Review by dcsaszar — APPROVED** [review]
Nice. Two suggestions.

**Comment by apepper** [comment]
REVIEW (54fe5d9)

**Review by dcsaszar — APPROVED** [review]

**Pull request merged**